user_docs/en/userGuide.t2t

```diff
@@ -1927,6 +1927,7 @@ Please see the display's documentation for descriptions of where these keys can
 | Move braille display to next line | t4 |
 | Scroll braille display forward | t5 or etouch3 |
 | Route to braille cell | routing |
+| Reports formatting under braille cell | secondRouting |

```

Comment:
Given the verbs above are toggle and route, I'd say this should be report instead of report.